File: abc/abc124/abc124_b.cs
// https://atcoder.jp/contests/abc124/submissions/8427640

using System;
using System.Linq;

namespace CSharp
{
    class Program
    {
        static void Main()
        {
            var N = int.Parse(Console.ReadLine());
            var H = Console.ReadLine().Split(' ').Select(long.Parse).ToArray();

            var result = 0;
            var t = H[0];
            for (int i = 0; i < N; i++)
            {
                if (H[i] >= t)
                {
                    result++;
                }
                t = Math.Max(H[i], t);
            }
            Console.WriteLine(result);
        }
    }
}

File: abc/abc124/abc124_b.fs
// https://atcoder.jp/contests/abc124/submissions/12208581

let N = stdin.ReadLine() |> int
let H = stdin.ReadLine().Split() |> Array.map int

let solver =
    let rec loop t r i =
        if i = N then r
        elif H.[i] >= t then loop (max H.[i] t) (r + 1) (i + 1)
        else loop (max H.[i] t) r (i + 1)
    loop H.[0] 0 0

solver |> stdout.WriteLine
